core/storage/base_model/gae_models.py

```diff
@@ -85,6 +86,10 @@
 class BaseModel(datastore_services.Model):
     """Base model for all persistent object storage classes."""
 
+    # Specifies whether the model's id is used as a key in Takeout. By default,
+    # the model's id is not used in Takeout.
```

Comment:
...is not used as the key for the takeout dict.

Reply:
done